Agent/Instrumentation/NSURLSession/NRMAURLSessionTaskOverride.m

```diff
@@ -126,7 +126,7 @@ void NRMAOverride__urlSessionTask_SetState(NSURLSessionTask* task, SEL _cmd, NSU
                 
                 NSURL *url = [currentRequest URL];
                 if (url != nil &&
-                    newState != NSURLSessionTaskStateRunning && task.state == NSURLSessionTaskStateRunning) {
```

Comment:
See slack convo, is this change really needed? The intent is to only call contents within if statement once per request.